.buildkite/longruns_gpu/pipeline.yml

```
@@ -94,7 +94,18 @@ steps:
           - "snowy_land_longrun_gpu/*pdf"
         agents:
           slurm_gpus: 1
-          slurm_time: 24:00:00
+          slurm_time: 15:00:00
         env:
           CLIMACOMMS_DEVICE: "CUDA"
           LONGER_RUN: ""
+
+      - label: "Soil"
```

```suggestion
      - label: "Very long run of soil"
```
To distinguish from the other soil longrun